### PR TITLE
Fix date when only one number used

### PR DESCRIPTION
**Pull request by petrican**

### CODE REVIEW

File: frontend/src/utils/documentListHelper.js [hunk 1]
```diff
@@ -347,6 +347,44 @@ export function getRowsData(rowData) {
   return data;
 }
 
+/**
+ * @method formatStringWithZeroSplitBy
+ * @summary For a given string with a specified separator it formats the number within with zeros in front if they are below 10
+ * @param {string} date
+ * @param {string} notation
+ */
+export function formatStringWithZeroSplitBy(date, notation) {
```
Comment (teosarca) [inline frontend/src/utils/documentListHelper.js:356]: i think u don't have to export `formatStringWithZeroSplitBy`.
does not look like it's used outside of this file.

File: frontend/src/utils/documentListHelper.js [hunk 2]
```diff
@@ -347,6 +347,44 @@ export function getRowsData(rowData) {
...
+  });
+  return frmArr.join(notation);
+}
+
+/**
+ * @method formatDateWithZeros
+ * @summary Format date with zeros if it's like dd.m.yyyy to dd.mm.yyyyy and similar for the case when / is the sepafrator
+ * @param {string} date
+ */
+export async function formatDateWithZeros(date) {
```
Comment (teosarca) [inline frontend/src/utils/documentListHelper.js:376]: pls provide unit tests for this function.

Comment (petrican) [reply frontend/src/utils/documentListHelper.js:376]: Done

File: frontend/src/utils/documentListHelper.js (hunk 1 above)
Comment (petrican) [reply frontend/src/utils/documentListHelper.js]: Yep. removed. Thanks for pointing this out.

**Pull request merged**